### PR TITLE
Combat

### PR DESCRIPTION
**Pull request by davidpodless**
There are a few problems I've notice here that need to be fixed and I'll be glad to hear your advice of how to do it

32. the left column, why there is a big space between the headline and the text?
33. there is a headline here that still in English, but in the file I did translate it, and the other parts of the same section it did took from the translation. so... what's the deal?
34. it looks like everything here just "escaped" out of the way it should have been

he-31
he-32
he-33
he-34
he-35
he-36

### CODE REVIEW

File: translations/combat.tex/he.po [hunk 1]
```diff
@@ -263,6 +369,17 @@ msgid ""
...
 "\n"
 msgstr ""
+"\\subsection*{\\pagetarget{CombatCards}{שימוש בקלפים במהלך הקרב}}\n"
+"ניתן להשתמש \\textbf{בקלף קסם אחד בלבד בכל סבב קרב}.\n"
+"ניתן להשתמש באפקטים מתמשכים \\includesvg[height=10px]{\\svgs/ongoing.svg} ו- \\includesvg[height=10px]{\\svgs/activation.svg} רק \\textbf{כאשר אחת היחידות שלך היא היחידה הפעילה ורק לפני שהיא תוקפת}.\n"
+"אפקטים מתמשכים \\index{Ongoing Effects} נשארים עד לסוף הקרב או עד שנגמרו השימושים בקלף.\\par\n"
+"ניתן לשחק בקלפים מידיים \\includesvg[height=10px]{\\svgs/instant.svg} \\textbf{בכל רגע נתון} למעט בין זריקת קובית התקיפה ובין הורדת נקודות הפגיעה, אלא אם כן צויין אחרת.\n"
+"קלפים המחזקים את ה- \\includesvg[height=10px]{\\svgs/attack.svg} של היחידה (למשל קלפי סטטיסטיקה), מסתיימים בהתקפה הראשונה של היחידה, או כאשר ההפעלה שלה מסתיימת, המוקדם מביניהם.\n"
+"חיזוק ה-\\includesvg[height=10px]{\\svgs/defense.svg} מסתיים באופן דומה.\n"
+"\\subsection*{\\pagetarget{Timelimit}{Combat Time Limits}}\\index{Time Limit}\n"
```
Comment (DarthGandalf) [inline translations/combat.tex/he.po:379]: This is the English string

**Comment by DarthGandalf** [comment]
Something is wrong with the translation - why is the hourglass image there twice?

File: translations/combat.tex/he.po [hunk 2]
```diff
@@ -410,6 +581,8 @@ msgid ""
 "The defender also gains the Arrow Tower\\index{Arrow Tower} Unit Card which is placed next to the Combat Board.\n"
 "The attacker doesn't need to destroy it to win the Combat."
 msgstr ""
+"המגן יכול גם להשתמש בקלף מגדל החצים \\textenglish{(Arrow Tower)}\\index{Arrow Tower} המנוח ליד לוח הקרב.\n"
```
Comment (qwrtln) [inline translations/combat.tex/he.po:584]: Missing arrow tower graphic

**Comment by qwrtln** [comment]
>     32. the left column, why there is a big space between the headline and the text?
I can't really tell which part of the Hebrew text is this, but try putting `\vspace*{\fill}` at the end of `itemize`. It should move everything up.

>     34. it looks like everything here just "escaped" out of the way it should have been
I think that's because you were in the middle of translating while the restructuring occurred. Some stuff was moved around only partially. Maybe starting on a fresh po file without translations would help? Then check if the text matches. If it does, copy it. If not, adjust it. Like @DarthGandalf, hourglass appears now twice, so something is clearly wrong.

**Comment by davidpodless** [comment]
I've reworked this section

it looks like the problems in the graphic that we see is because Hebrew is "shorter" language and the graphics assume we have more text than we actually have
he-31
he-32
he-33
he-34
he-35
he-36

**Comment by qwrtln** [comment]
Yes, well, this one is tough. Combat is long enough for Hebrew to make it one page shorter. However, this will break the layout and won't pass the checks. I'm not sure how to best approach this.

**Comment by qwrtln** [comment]
The layout was changed recently. Can you rebase and build it again? I wonder if it fixes your issue.

**Comment by qwrtln** [comment]
@davidpodless Are you planning to fix this?

**Comment by davidpodless** [comment]
I have no idea how

So in the meantime I'm doing everything else and then I'll come back to it

**Comment by davidpodless** [comment]
OK, Im closing this pull request and Im opening a new one

**Pull request closed without merging**